lib/ansible/modules/network/iosxr/iosxr_config.py

```diff
@@ -260,7 +261,8 @@ def run(module, result):
 
             result['updates'] = commands
 
-        load_config(module, commands, result)
+        load_config(module, commands, not module.check_mode)
```

Comment:
yep, cargo-culted from module_utils/ios.py which didn't have 'comment', will push a patch to fix it as well.